### PR TITLE
Add four new repos

### PR DESCRIPTION
**Pull request by aleron75**
This PR changes Terraform's `variables.tf` file to add the following new reports, as discussed in [this tech meeting](https://docs.google.com/document/d/1t60o_LL_a-SBQkb24cc7g2vcZhV8efZObiwtJJFVVl0/edit#):

- Magento Cloud Patches (mirror)
- Magento Quality Patches Tool (mirror)
- Magento Asynchronous Events (port)
- Meta (for documents, guidelines, etc.)

### CODE REVIEW

**Comment by damienwebdev** [comment]
@aleron75 your PR failed because of:

```bash
terraform format -check -diff
```

exited with status `3`

This is just a document linter verifying that you nicely formatted your `tf` files.

Try grabbing terraform locally and running this command.

https://developer.hashicorp.com/terraform/downloads

**Comment by aleron75** [comment]
Thank you @damienwebdev it worked!

File: variables.tf [hunk 1]
```diff
@@ -186,6 +186,16 @@ variable "repositories" {
       teams       = ["infrastructure"]
     }
 
+    mage-os-mirror-magento-cloud-patches = {
```
Comment (damienwebdev) [inline variables.tf:189]: I think this goes into the `mirror_repositories` key above.

File: variables.tf [hunk 2]
```diff
@@ -186,6 +186,16 @@ variable "repositories" {
...
+      description = "This is a Mage-OS fork of the Magento Cloud Patches project found at https://github.com/magento/magento-cloud-patches."
+      teams       = ["infrastructure"]
+    }
+
+    mage-osmirror-quality-patches = {
```
Comment (damienwebdev) [inline variables.tf:194]: I think this goes into the `mirror_repositories` key above.

File: variables.tf (hunk 1 above)
Comment (damienwebdev) [inline variables.tf:189]: I think the key should be `mirror-magento-cloud-patches`

File: variables.tf (hunk 2 above)
Comment (damienwebdev) [inline variables.tf:194]: I think the key should be `mirror-quality-patches`

**Review by damienwebdev — CHANGES_REQUESTED** [review]

**Review by damienwebdev — DISMISSED** [review]

File: variables.tf [hunk 3]
```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento-cloud-patches = {
+      description = "This is a Mage-OS fork of the Magento Cloud Patches project found at https://github.com/magento/magento-cloud-patches."
+      teams       = ["infrastructure"]
```
Comment (sprankhub) [inline variables.tf:116]: All current mirror repositories have `continuous-integration` as a team. Why do the new ones have `infrastructure`?

File: variables.tf [hunk 4]
```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento-cloud-patches = {
```
Comment (sprankhub) [inline variables.tf:114]: I tried to sort the repositories alphabetically - would you mind updating it? :see_no_evil:

File: variables.tf [hunk 5]
```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
...
+      description = "This is a Mage-OS fork of the Magento Cloud Patches project found at https://github.com/magento/magento-cloud-patches."
```
Comment (sprankhub) [inline variables.tf:115]: Can we use the same format as the other mirror repositories? I would love to have them consistent...

**Review by sprankhub — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 3 above)
Comment (aleron75) [reply variables.tf:116]: You're right. We decided it during the call, but we likely made a mistake; I will correct it.

File: variables.tf (hunk 5 above)
Comment (aleron75) [reply variables.tf:115]: I guess here you are referring to the same thing I just asked: do you mean renaming it to `mirror-magento2-cloud-patches`?

Comment (aleron75) [reply variables.tf:115]: or maybe here you refer to converting "This is a" to "Mage-OS fork of..." 🤔

File: variables.tf (hunk 4 above)
Comment (sprankhub) [reply variables.tf:114]: I think the current naming scheme is `mirror-{{ORIGINAL_REPOSITORY_NAME}}`.

File: variables.tf (hunk 5 above)
Comment (sprankhub) [reply variables.tf:115]: Here I am referring to the description. I do not care too much about them, but I think they should all have the same format :)

**Comment by aleron75** [comment]
@sprankhub @damienwebdev in my last commit (b0c43569913d6b29ced6ee8bbf8232b724375896), I took the chance to fix sorting for everything

**Review by sprankhub — APPROVED** [review]

**Pull request merged**